packages/web-components/src/player-component/player.class.ts

```diff
@@ -332,12 +331,12 @@ export class PlayerWrapper {
 
         // Remove timeline
         this.removeTimelineComponent();
-        // await this.controls.destroy();
 
         if (this.isLoaded) {
             this.video.pause();
             this.video.src = '';
             await this.player?.unload();
+            await this.controls.destroy();
```

Comment:
Maybe this.controls = undefined for extra safety?

Reply:
done